content/docs/0.8.1/volumes-and-nodes/create-volumes.md

```diff
@@ -54,4 +86,34 @@ The above yaml file contains two parts:
         persistentVolumeClaim:
           claimName: longhorn-volv-pvc
     ```
-More examples are available [here.](../../users-guide/examples)
\ No newline at end of file
+More examples are available [here.](../../users-guide/examples)
+
+### Binding Workloads to PVs without a Kubernetes StorageClass
+
+It is possible to use a Longhorn StorageClass to bind a workload to a PV without creating a StorageClass object in Kubernetes. 
+
+In this scenario, only an existing PV is used.
+
+PVC asked for a non-exist storageclass (means no storageclass object and no PV has that storageclass), PVC will be pending
```

Comment:
`StorageClass`. Also missing the dot.

Reply:
This section had some unfinished notes that I forgot about. Here's the revised section:

> It is possible to use a Longhorn StorageClass to bind a workload to a PV without creating a StorageClass object in Kubernetes. 
> 
> Since the Storage Class is also a field used to match a PVC with a PV, which doesn't have to be created by a Provisioner, you can create a PV manually with a custom StorageClass name, then create a PVC asking for the same StorageClass name. Kubernetes will bind your PVC with your PV with the same StorageClass name, even if the StorageClass resource doesn't exist. 
> 
> When a PVC requests a StorageClass that does not exist as a Kubernetes resource, the PVC will be pending. The StorageClass will be used like a label to find the matching PV, and only existing PVs labeled with the StorageClass name will be used.
> 
> If the PVC names a StorageClass, Kubernetes will:
> 
> 1. Look for an existing PV that has the label matching the StorageClass
> 2. Look for an existing StorageClass Kubernetes resource. If the StorageClass exists, it will be used to create a PV.